File: app/src/main/java/com/coppermobile/mysampleusingmvvmlivedata/data/source/remote/DishResponse.java
package com.coppermobile.mysampleusingmvvmlivedata.data.source.remote;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class DishResponse implements Serializable{

    @SerializedName("thumbnail")
    private String thumbnail;

    @SerializedName("price")
    private int price;

    @SerializedName("name")
    private String name;

    @SerializedName("description")
    private String description;

    @SerializedName("id")
    private int id;

    public String getThumbnail() {
        return thumbnail;
    }

    public int getPrice() {
        return price;
    }

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public int getId() {
        return id;
    }
}

File: app/src/main/java/com/coppermobile/kotlinsamplemvvmlivedataroom/data/source/remote/DishResponse.kt
package com.coppermobile.kotlinsamplemvvmlivedataroom.data.source.remote

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class DishResponse : Serializable {

    @SerializedName("thumbnail")
    var thumbnail: String? = null
        private set

    @SerializedName("price")
    var price: Int? = -1
        private set

    @SerializedName("name")
    var name: String? = null
        private set

    @SerializedName("description")
    var description: String? = null
        private set

    @SerializedName("id")
    var id: Int? = -1
        private set
}